test/functional/apps/visualize/_vertical_bar_chart.js

```diff
@@ -273,6 +274,20 @@ export default function ({ getService, getPageObjects }) {
         expect(legendEntries).to.eql(expectedEntries);
       });
 
+      it('should show an error if last bucket aggregation is terms', async () => {
+        await PageObjects.visualize.toggleOpenEditor(2, 'false');
+        await PageObjects.visualize.clickAddBucket();
+        await PageObjects.visualize.clickBucket('Split Series');
+        await PageObjects.visualize.selectAggregation('Terms');
+        await PageObjects.visualize.selectField('response.raw');
+        await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
Actually, It's better to use `await renderable.waitForRender();` to wait for the rendering completion that checks for `data-render-complete` and `data-loading` flags